app/templates/complaint-form.hbs

```diff
@@ -234,7 +234,7 @@
           class="au-u-margin-bottom-small full-width"
           @filename={{attachment.filename}}
           @fileSize={{attachment.humanReadableSize}}
-          @onDelete={{this.deleteFile}}
+          @onDelete={{fn this.deleteFile attachment}}
```

Comment:
I think the removal was broken in the previous version since we weren't passing the record into the action yet.